boards/sma_q3/src/main.rs

```diff
@@ -433,6 +437,53 @@ pub unsafe fn start() -> (
     );
     periodic_virtual_alarm.setup();
 
+    let screen = {
+        let mux_spi = components::spi::SpiMuxComponent::new(&base_peripherals.spim2)
+            .finalize(components::spi_mux_component_static!(nrf52840::spi::SPIM));
+
+        use kernel::hil::spi::SpiMaster;
+        base_peripherals
+            .spim2
+            .set_rate(1_000_000)
+            .expect("SPIM2 set rate");
+
+        base_peripherals.spim2.configure(
+            nrf52840::pinmux::Pinmux::new(Pin::P0_27 as u32),
+            nrf52840::pinmux::Pinmux::new(Pin::P0_28 as u32),
+            nrf52840::pinmux::Pinmux::new(Pin::P0_26 as u32),
+        );
+
+        let disp_pin = &nrf52840_peripherals.gpio_port[Pin::P0_07];
+        let cs_pin = static_init!(
+            components::lpm013m126::Inverted<'static, nrf52840::gpio::GPIOPin>,
+            components::lpm013m126::Inverted(&nrf52840_peripherals.gpio_port[Pin::P0_05])
+        );
+
+        let display = components::lpm013m126::Lpm013m126Component::new(
+            mux_spi,
+            cs_pin,
+            disp_pin,
+            &nrf52840_peripherals.gpio_port[Pin::P0_06],
+            mux_alarm,
+        )
+        .finalize(components::lpm013m126_component_static!(
+            nrf52840::rtc::Rtc<'static>,
+            nrf52840::gpio::GPIOPin,
+            nrf52840::spi::SPIM
+        ));
+
+        let screen = components::screen::ScreenComponent::new(
+            board_kernel,
+            capsules_extra::screen::DRIVER_NUM,
+            display,
+            None,
+        )
+        .finalize(components::screen_component_static!(4096));
+        // Power on screen if not already powered
+        let _ = display.set_power(true);
```

Comment:
Should we let userspace or whatever is driving the screen do this instead?

Reply:
Perhaps? It's not well supported by the user space drivers now, so possibly to consider in a different change